react/components/EXPERIMENTAL_useCheckboxTree/constants.ts

```diff
@@ -1,5 +1,10 @@
-export const defaultComparatorCurry = (item: any) => (candidate: any) =>
-  item.id === candidate.id
+type itemWithID = {
```

Comment:
```suggestion
type ItemWithID = {
```
(But just suggestion)

Reply:
Accepted